### ISSUE TITLE
Some problems I encountered when using java to connect tigase

### ISSUE THREAD

**Issue by fullky** [tigase/tigase-server#124]
**I have a problem with…**
Smack I use;Error Message.
![image](https://user-images.githubusercontent.com/91643533/143370850-87e8b3f3-39a5-4e3f-9359-0cd5d47eb564.png)

Do I need a certificate? If so, is that tigase's certificate? Can I connect to tigase without a certificate?

**Details (please complete the following information):**
 - Tigase version: [e.g. 8.1.2]
 - JVM flavour and version [e.g. AdoptOpenJDK11]
 - Operating system/distribution/version [e.g. Linux centos7.8]

**Comment by woj-tek** [comment]
Yes, you need certificate that matched domain/Vhost that you've configured in the Tigase. tigase-server should automatically generate self-signed certificate but those are not considered secure enough thus most of the libraries/clients reject them by default (some give you option to accept those but it depends on the client/library).

Please check our documentation: [Server Certificates](https://docs.tigase.net/tigase-server/master-snapshot/Administration_Guide/html/#ServerCertificates)
